### PR TITLE
Added hello-world app in NodeJS

### PR DESCRIPTION
**Pull request by somishraredhat**
Hi Shatakshi,

I have created pull request for hello world app. Please go through.

@psolarvi ,
Please take a look if you have time.

Thanks.

### CODE REVIEW

File: serverless-hello-world/func.yaml
Comment (psolarvi) [inline serverless-hello-world/func.yaml:4]: At some point we need to move this image to the RedHatTraining namespace.

File: serverless-hello-world/index.js
Comment (psolarvi) [inline serverless-hello-world/index.js:21]: Why we want to make the application to launch an error on other methods than GET/POST?

If we decide to avoid throwing errors, perhaps we can tweak the code and:
- Keep the first if that returns the 200 on the GET
- Remove the second if and return always 405.

That way we only return 200 in the GET. In any other cases, the if is not fired, so the return of 405 is what is going to happen.

File: serverless-hello-world/index.js
Comment (psolarvi) [inline serverless-hello-world/index.js:24]: I think that the module.exports line is self-explanatory so perhaps we can remove this comment.

File: serverless-hello-world/package.json
Comment (psolarvi) [inline serverless-hello-world/package.json:2]: Should we keep this name in sync with the application name (hello-world)?

File: serverless-hello-world/package.json
Comment (psolarvi) [inline serverless-hello-world/package.json:13]: Looks like we are defining the license as Apache 2, but there is no license file in the PR?

File: serverless-hello-world/test/test.js
Comment (psolarvi) [inline serverless-hello-world/test/test.js:31]: The test only cover GET/POST, what happens with other methods such as PUT?

**Review by psolarvi — COMMENTED** [review]
Nice PR! I added no major comments, I didn't have the time to test it, but please don't wait for my approval. We can always tweak later the app.

BTW I should tweak the PR name to something like: "Added hello-world app in NodeJS" or something similar.

**Comment by shatakshi06** [comment]
> Hi Shatakshi,
> 
> I have created pull request for hello world app. Please go through.
> 
> @psolarvi , Please take a look if you have time.
> 
> Thanks.

Sure @somishraredhat

**Review by shatakshi06 — COMMENTED** [review]
Hi @somishraredhat 

I have gone through the Hello World application. 

Thanks,
Shatakshi

File: serverless-hello-world/func.yaml
Comment (somishraredhat) [reply serverless-hello-world/func.yaml:4]: I will remove the image name. That got here by mistake. By default it should ask for the name of image when you build it for first time.

File: serverless-hello-world/index.js
Comment (somishraredhat) [reply serverless-hello-world/index.js:21]: Done

File: serverless-hello-world/index.js
Comment (somishraredhat) [reply serverless-hello-world/index.js:24]: Done

**Comment by somishraredhat** [comment]
Hi @psolarvi ,

I added changes recommended by you. Please check and approve.
Thanks

**Review by psolarvi — APPROVED** [review]

**Pull request merged**
